transports/quic/src/config.rs

```diff
@@ -63,6 +63,9 @@ pub struct Config {
     server_tls_config: Arc<rustls::ServerConfig>,
     /// Libp2p identity of the node.
     keypair: libp2p_identity::Keypair,
+
+    /// Parameters governing MTU discovery. See [`MtuDiscoveryConfig`] for details.
+    path_mtu_discovery_config: Option<MtuDiscoveryConfig>,
```

Comment:
```suggestion
    mtu_discovery_config: Option<MtuDiscoveryConfig>,
```

Nit pick to be consistent with the `quinn` type.